pkgs/top-level/all-packages.nix

```diff
@@ -8490,7 +8490,14 @@ in
 
   inherit (haskellPackages) ghc;
 
-  cabal-install = haskell.lib.justStaticExecutables haskellPackages.cabal-install;
+  cabal-install = with haskell.lib; overrideCabal (justStaticExecutables haskellPackages.cabal-install) (drv: {
+    executableToolDepends = [ makeWrapper ];
+    postInstall = ''
+      wrapProgram $out/bin/cabal \
+        --prefix PATH : ${lib.makeBinPath [ binutils-unwrapped haskellPackages.ghc ]}
```

Comment:
I don't want cabal-install referencing that compiler since I use a different one most of the time. On some machines I even use a compiler that wasn't installed by Nix at all.

Reply:
Ok, removed ghc again